WeakAurasTemplates/TriggerTemplatesData.lua

```diff
@@ -4209,15 +4586,12 @@ end
 -- PVP Talents
 -------------------------------
 
---[[
+
 for _, class in pairs(templates.class) do
   for _, spec in pairs(class) do
-  -- TODO 8.0
-  -- tinsert(spec[5].args, { spell = 195710, type = "ability"}) -- Honorable Medallion
-  -- tinsert(spec[5].args, { spell = 208683, type = "ability", pvptalent = 1}) -- Gladiator's Medallion
+    tinsert(spec[7].args, { spell = 208683, type = "ability", pvptalent = 1}) -- Gladiator's Medallion
```

Comment:
This doesn't test if `spec[7]` exists which make an error with current master for templates.class.ROGUE.classic